demo/src/main/java/fj/demo/Option_filter.java

```diff
@@ -19,12 +19,12 @@ public static void main(final String[] args) {
         final Option<Integer> o6 = o3.filter(even);
 
         F<Integer, Boolean> f = i -> i % 2 == 0;
-        final Option<Integer> o7 = o1.filter(f);
-        final Option<Integer> o8 = o1.filter(f);
-        final Option<Integer> o9 = o1.filter(i -> i % 2 == 0);
+        final Option<Integer> o7 = o4.filter(f);
+        final Option<Integer> o8 = o5.filter(f);
+        final Option<Integer> o9 = o6.filter(i -> i % 2 == 0);
 
-        optionShow(intShow).println(o4); // None
-        optionShow(intShow).println(o5); // None
-        optionShow(intShow).println(o6); // Some(8)
+        optionShow(intShow).println(o7); // None
+        optionShow(intShow).println(o8); // None
+        optionShow(intShow).println(o9); // Some(4)
```

Comment:
Hi @samekmichal, can you change the comment here to Some(8).  It filters on the even condition without changing the value of the some.